apps/artist/templates/page.jade

```diff
@@ -17,7 +17,7 @@ block content
         | #{artist.get('forsale_artworks_count')} for sale
       #artist-page-sort
         label.garamond-select: select( dir="rtl" )
-          for val, label in { "Sort by relevance": '', "Sort by recently added": '-published_at' }
+          for val, label in { "Recently Updated": "-partner_updated_at", "Recently added": '-published_at', "Artwork year - asc": "year" , "Artwork year - desc": '-year' }
```

Comment:
Are we showing “asc” and “desc” to the user in the UI?

@briansw How do you feel about using “↓” and “↑” instead?